src/tgen-transport.c

```diff
@@ -1121,6 +1121,38 @@ static TGenEvent _tgentransport_receiveSocksResponseStatus(TGenTransport* transp
                         g_string_append_printf(messageBuffer, "address type not supported");
                         break;
                     }
+                    case 0xF0: {
```

Comment:
If we connect to a non-Tor SOCKS proxy (which I realize is unlikely given tgen's main purpose), is it possible that they send us these error codes but they have a different meaning? Might be worth a comment in the code?